### DISCUSSION TITLE
Support Typanion

### DISCUSSION THREAD

**Discussion by stramel** [react-hook-form/react-hook-form#6099] [Ideas]
An alternative to Yup is Typanion which is a typesafe, tree-shakable, functional api lib. 

https://github.com/arcanis/typanion

(Yarn v3 switched from `yup` to `typanion`)

**Comment by bluebill1049** [comment]
interesting 👍 Do you want to send a PR @stramel?

**Reply by stramel** [reply]
Yeah, i can whip one up tomorrow

**Reply by bluebill1049** [reply]
Awesome!
